spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/ArrowBinaryIterator.java

```diff
@@ -50,7 +52,8 @@ public ArrowBinaryIterator(
       List<String> columnsInOrder,
       ByteString schema,
       ByteString rowsInBytes,
-      Optional<StructType> userProvidedSchema) {
+      Optional<StructType> userProvidedSchema,
+      Optional<BigQueryStorageReadRowsTracer> bigQueryStorageReadRowsTracer) {
```

Comment:
unused?

Reply:
rowParseFinished is called on tracer on line 113